packages/graphql-hooks-memcache/README.md

```diff
@@ -0,0 +1,42 @@
+# graphql-hooks-memcache
+
+In-memory caching implementation for graphql-hooks
+
+## Install
+
+`npm install graphql-hooks-memcache`
+
+or
+
+`yarn add graphql-hooks-memcache`
+
+## Quick Start
+
+This is intended to be used as the `cache` option when calling `createClient` from `graphql-hooks`.
+
+```js
+import { createClient } from 'graphql-hooks';
+import memCache from 'graphql-hooks-memcache';
+
+const client = createClient({
```

Comment:
This is the only change in `v1.0.5` - I updated the docs to the new api.

Reply:
I've updated this package to to v1.0.5 in https://github.com/nearform/graphql-hooks/pull/78/commits/e047a50552cb310705d275e00f2d0b5dbd5a0db9